tox.ini

```diff
@@ -11,46 +11,39 @@ usedevelop = True
 install_command = pip install {opts} {packages}
 
 deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
+       -r{toxinidir}/requirements-test.txt
 commands =
   pytest gitlab/tests {posargs}
 
 [testenv:pep8]
 basepython = python3
-deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
-       flake8
+deps = -r{toxinidir}/requirements-lint.txt
 commands =
   flake8 {posargs} .
 
 [testenv:black]
 basepython = python3
-deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
-       black
+deps = -r{toxinidir}/requirements-lint.txt
 commands =
   black {posargs} .
 
 [testenv:isort]
 basepython = python3
-deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
-       isort
+deps = -r{toxinidir}/requirements-lint.txt
 commands =
   isort --dont-order-by-type {posargs} {toxinidir}
 
 [testenv:twine-check]
 basepython = python3
 deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
+       -r{toxinidir}/requirements-test.txt
        twine
 commands =
   twine check dist/*
 
 [testenv:mypy]
 basepython = python3
-deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
+deps = -r{toxinidir}/requirements-lint.txt
```

Comment:
If going to do this then we might want to use a common tox env dir. Then they can share the same space.

```
envdir={toxworkdir}/pep8
```